src/sql/workbench/parts/query/common/queryInput.ts

```diff
@@ -117,7 +118,8 @@ export class QueryInput extends EditorInput implements IEncodingSupport, IConnec
 		private _connectionProviderName: string,
 		@IConnectionManagementService private _connectionManagementService: IConnectionManagementService,
 		@IQueryModelService private _queryModelService: IQueryModelService,
-		@IConfigurationService private _configurationService: IConfigurationService
+		@IConfigurationService private _configurationService: IConfigurationService,
+		@IExtensionTipsService private _extensionTipsService: IExtensionTipsService
```

Comment:
Can you revert changes to this file? You're not using this constructor parameter anywhere